docs/source/conf.py

```diff
@@ -322,7 +322,8 @@
     "Quantize",
     # torch.utils.backcompat
     "Warning",
-    "SymIntNode"
+    "SymIntNode",
```

Comment:
There should really either have a doc entry on the website or be private...

Reply:
It should get doc'ed. But TBH, I would prefer we call these SymInt in Python and not SymIntNode. (More comments from the peanut gallery? @Chillee @Krovatkin )

Reply:
I'd rather we do this in a separate PR, there's a bunch more doc stuff we'll have to do to make all of this make sense.